### ISSUE TITLE
Search

### ISSUE THREAD

**Issue by alekzuz** [izzyumkin/EmojiPicker#25]
Please add search field to search emoji

**Comment by izyumkin** [comment]
Yes, of course I will add it! I'm already done with collecting logalized strings for all languages for search. 

Now I'm working on the UI part, it has its own difficulties to make a perfect resemblance to the macOS version.

I'll let you know when it's ready🙏

**Comment by blubber** [comment]
Hey @izyumkin I'm looking to use your emoji picker in an app that I'm currently working on, but the lack of a search option is a deal breaker for me. Just wondering if you're making any progress on this and if I can lend you a hand.

**Comment by 0xifarouk** [comment]
Any progress on this please? @izyumkin

**Comment by findroli** [comment]
Another potential user here @izyumkin - but the search functionality is really crucial! 🥲 Great job you've done though 🙌

**Comment by mapierce** [comment]
Hey I've got a PR open for this https://github.com/izyumkin/MCEmojiPicker/pull/57
